tests/aws/services/route53resolver/test_route53resolver.py

```diff
@@ -721,3 +722,221 @@ def test_list_firewall_domain_lists(self, cleanups, snapshot, aws_client):
+            Priority=1,
+            Action=Action.ALLOW,
+            Name=f"rule-name-{short_uid()}",
+        )
+        snapshot.match("create-firewall-rule-1", rule_response_1)
+        cleanups.append(
+            lambda: aws_client.route53resolver.delete_firewall_rule(
+                FirewallRuleGroupId=rule_group_response["FirewallRuleGroup"]["Id"],
+                FirewallDomainListId=domain_list_response_1["FirewallDomainList"]["Id"],
+            )
+        )
+
+        rule_response_2 = aws_client.route53resolver.create_firewall_rule(
+            FirewallRuleGroupId=rule_group_response["FirewallRuleGroup"]["Id"],
+            FirewallDomainListId=domain_list_response_2["FirewallDomainList"]["Id"],
+            Priority=2,
+            Action=Action.ALERT,
+            Name=f"rule-name-{short_uid()}",
+        )
+        snapshot.match("create-firewall-rule-2", rule_response_2)
+        cleanups.append(
+            lambda: aws_client.route53resolver.delete_firewall_rule(
+                FirewallRuleGroupId=rule_group_response["FirewallRuleGroup"]["Id"],
+                FirewallDomainListId=domain_list_response_2["FirewallDomainList"]["Id"],
+            )
+        )
+
+        rule_response_3 = aws_client.route53resolver.create_firewall_rule(
+            FirewallRuleGroupId=rule_group_response["FirewallRuleGroup"]["Id"],
+            FirewallDomainListId=domain_list_response_3["FirewallDomainList"]["Id"],
+            Priority=3,
+            Action=Action.ALERT,
+            Name=f"rule-name-{short_uid()}",
+        )
+        snapshot.match("create-firewall-rule-3", rule_response_3)
+        cleanups.append(
+            lambda: aws_client.route53resolver.delete_firewall_rule(
+                FirewallRuleGroupId=rule_group_response["FirewallRuleGroup"]["Id"],
+                FirewallDomainListId=domain_list_response_3["FirewallDomainList"]["Id"],
+            )
+        )
+
+        rule_response_4 = aws_client.route53resolver.create_firewall_rule(
+            FirewallRuleGroupId=rule_group_response["FirewallRuleGroup"]["Id"],
+            FirewallDomainListId=domain_list_response_4["FirewallDomainList"]["Id"],
+            Priority=4,
+            Action=Action.ALLOW,
+            Name=f"rule-name-{short_uid()}",
+        )
+        snapshot.match("create-firewall-rule-4", rule_response_4)
+        cleanups.append(
+            lambda: aws_client.route53resolver.delete_firewall_rule(
+                FirewallRuleGroupId=rule_group_response["FirewallRuleGroup"]["Id"],
+                FirewallDomainListId=domain_list_response_4["FirewallDomainList"]["Id"],
+            )
+        )
+
+        # Check list filtering
+        list_all_response = aws_client.route53resolver.list_firewall_rules(
+            FirewallRuleGroupId=rule_group_response["FirewallRuleGroup"]["Id"]
+        )
+
+        assert len(list_all_response["FirewallRules"]) == 4, "Expecting 4 firewall rules"
```

Comment:
We can skip this `assert` since the snapshot will already capture this response.

Reply:
Done.